product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx

```diff
@@ -576,3 +491,63 @@ bdr.wait_for_join_completion(verbose_progress boolean DEFAULT false)
+| `check_full_tuple` | Reserved for backward compatibility. |
+| `apply_delay` | How long nodes wait to apply incoming changes. This is useful mainly to set up a special sub-group with delayed subscriber-only nodes. Don't set this on groups that contain data nodes or on the top-level group. Default is `0s`. |
+| `check_constraints` | Whether the apply process checks the constraints when writing replicated data. It's recommended to keep this to the default value, otherwise you risk data loss. Valid values are either `on` or `off`. Default is `on`. |
+| `num_writers` | Number of parallel writers for the subscription backing this node group. Valid values are either `-1` or a positive integer. `-1` means the value specified by the GUC `bdr.writers_per_subscription` is used. `-1` is the default. |
+| `enable_wal_decoder` | Enables/disables the decoding worker process. You can't enable the decoding worker process if `streaming_mode` is already enabled. Valid values are either `on` or `off`. Default is `off`. |
+| `streaming_mode` | Enables/disables streaming of large transactions. When set to `off`, streaming is disabled. When set to any other value, large transactions are decoded while they're still in progress, and the changes are sent to the downstream. If the value is set to `file`, then the incoming changes of streaming transactions are stored in a file and applied only after the transaction is committed on upstream. If the value is set to `writer`, then the incoming changes are directly sent to one of the writers, if available. If parallel apply is disabled or no writer is free to handle streaming transaction, then the changes are written to a file and applied after the transaction is committed. If the value is set to `auto`, PGD tries to intelligently pick between `file` and `writer`, depending on the transaction property and available resources. You can't enable `streaming_mode` if the WAL decoder is already enabled.<br/><br/>For more details, see [Transaction streaming](../transaction-streaming). |
+| `default_commit_scope` | The commit scope to use by default, initially the `local` commit scope. This applies only to the top-level node group. You can use individual rules for different origin groups of the same commit scope. See [Origin groups](../durability/commit-scopes/#origin-groups) for more details. |
+
+### Notes
+
+This function passes a request to the group consensus mechanism to change
+the defaults. The changes made are replicated globally using the consensus
+mechanism.
+
+The function isn't transactional. The request is processed in the background
+so you can't roll back the function call. Also, the changes might not be
+immediately visible to the current transaction.
+
+This function doesn't hold any locks.
+
+!!! Warning
+    When you use this function to change the `apply_delay` value, the
+    change doesn't apply to nodes that are already members of the
+    group.
```

Comment:
I think changes do apply to nodes that already are members of the group.

Reply:
warning removed